doc/topics/packaging/index.rst

```diff
@@ -23,11 +91,40 @@ You only need to run rpmbuild in the Salt repo:
 How to build deb packages
 =========================
 
-You only need to add a symlink and run debuild in the Salt repo:
+Install the dependencies:
```

Comment:
**Suggestion (style, blocking):** Since this is a task that you're describing, each of the steps should be formatted in an ordered list.

For optional or conditional steps, you could say something like:

```
#. (Optional) To build a specific version of Salt.
```

Or if there are multiple options, you could list it as sub-bullet points.